tests/docker/start.irods.ubuntu22.sh

```diff
@@ -1,4 +1,5 @@
 #! /bin/bash
+#apt install -y /irods_packages/irods-database-plugin-postgres_4.3.3-0~jammy_amd64.deb /irods_packages/irods-icommands_4.3.3-0~jammy_amd64.deb /irods_packages/irods-runtime_4.3.3-0~jammy_amd64.deb /irods_packages/irods-server_4.3.3-0~jammy_amd64.deb
```

Comment:
Is this needed?

Reply:
Yes, this is the line that would be uncommented and modified to install a locally built version.  I've updated the README.md to include these instructions.  I missed these README steps for the iRODS server.

Reply:
Avoid use of `apt` in scripts. Instead, `apt-get` should be used. This applies to all usages of `apt`.

Reply:
Changed all apt to apt-get.